src/traces/scattermapbox/plot.js

```diff
@@ -50,23 +54,40 @@ proto.setSourceData = function(k, opts) {
         .setData(opts.geojson);
 };
 
-proto.addLayer = function(k, opts) {
-    this.subplot.map.addLayer({
+proto.addLayer = function(k, opts, below) {
+    this.subplot.addLayer({
         type: k,
         id: this.layerIds[k],
         source: this.sourceIds[k],
         layout: opts.layout,
         paint: opts.paint
-    });
+    }, below);
 };
 
 proto.update = function update(calcTrace) {
     var subplot = this.subplot;
+    var map = subplot.map;
     var optsAll = convert(calcTrace);
+    var below = subplot.belowLookup['trace-' + this.uid];
+    var i, k, opts;
+
+    if(below !== this.below) {
+        // console.log('update below', [below, this.below])
```

Comment:
Thought you may want to remove this line.

Reply:
Good eye. Thanks!

https://github.com/plotly/plotly.js/pull/4058/commits/39eff3a03cc76729d3b0163e09638e8b16b4f8be